site/content/en/docs/manual/advanced/skeletons.md

```diff
@@ -0,0 +1,216 @@
+  - [Adding Skeleton manually](#adding-skeleton-manually)
+  - [Adding Skeleton labels from the model](#adding-skeleton-labels-from-the-model)
+  - [Configuring Skeleton points](#configuring-skeleton-points)
+- [Annotation with Skeletons](#annotation-with-skeletons)
+- [Automatic annotation with Skeletons](#automatic-annotation-with-skeletons)
+- [Editing skeletons on the canvas](#editing-skeletons-on-the-canvas)
+- [Editing skeletons on the sidebar](#editing-skeletons-on-the-sidebar)
+
+## Skeleton task
+
+To start annotating using skeletons, you need to set up a **Skeleton** task
+in **Configurator**:
+
+To open **Configurator**, when [creating a task](/docs/manual/basics/create_an_annotation_task/),
+click on the **Setup skeleton** button if you want to set up the skeleton manually,
+or **From model** if you want to add skeleton labels from the model.
+
+![](/images/image-setup-skeleton-1.jpg)
+
+The skeleton **Configurator** has the following fields:
+
+![](/images/image-skeleton-configurator-overview.jpg)
+
+<!--lint disable maximum-line-length-->
+
+| Number | Name                        | Description                                                                                                                           |
+| ------ | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| **1**  | **Upload background image** | (Optional) Use it to upload a background image, to draw a skeleton on top of it.                                                      |
+| **2**  | **Add point**               | Use it to add Skeleton points to the **Drawing area** (**8**).                                                                        |
+| **3**  | **Click and drag**          | Use it to move points across the **Drawing area** (**8**).                                                                            |
+| **4**  | **Add edge**                | Use it to add edge on the **Drawing area** (**8**) to connect the points (**2**).                                                     |
+| **5**  | **Remove point**            | Use it to remove points. Click on **Remove point** and then on any point (**2**) on the **Drawing area** (**8**) to delete the point. |
+| **6**  | **Download skeleton**       | Use it to download created skeleton in .SVG format.                                                                                   |
+| **7**  | **Upload skeleton**         | Use it to upload skeleton in .SVG format.                                                                                             |
+| **8**  | **Drawing area**            | Use it as a canvas to draw a skeleton.                                                                                                |
+
+<!--lint enable maximum-line-length-->
+
+You can add **Skeleton** points manually or use the model.
+
+### Adding Skeleton manually
```

Comment:
I feel like Skeleton configurator should be described exactly in this section, not above